.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "stylelint.vscode-stylelint",
+        "esbenp.prettier-vscode",
+        "dbaeumer.vscode-eslint",
+        "streetsidesoftware.code-spell-checker",
+        "astro-build.astro-vscode"
+      ]
+    }
+  },
+  "updateContentCommand": "yarn install"
```

Comment:
More improvements: can we add [port attributes](https://containers.dev/implementors/json_reference/#port-attributes)?

I'm thinking it would be useful to have labels for all forwarded ports. e.g. 5173 is "css-workshop", 6006 is "react-workshop", etc

Also, running `yarn dev` opens the portal (port 3000) in a whole new window, which is (1) jarring, (2) useless because clicking on any of them does not open the correct thing. I believe `onAutoForward` can be set to `silent` to prevent it opening a new tab?

Reply:
added those

Reply:
3000 is still opening in a new tab. Can you try `"ignore"` instead of `"silent"`?

Reply:
yeah ill try that

Reply:
It's still not working as expected using `"silent"`, but let's look into it at a different point in the future.